### PR TITLE
feat: complete websocket implementation for swipe sessions and fix edge cases

### PR DESCRIPTION
**Pull request by wwz2003**
- Added swipe_session_update broadcast to Daphne consumer and views.
- Wired frontend app-context to automatically fetch swipe session progress on websocket update.
- Updated swipe page to display 'Back to Group' instead of 'Back to Home'.
- Handled member removal edge cases to ensure active swipe sessions calculate matches correctly.

### CODE REVIEW

File: backend/groups/tests.py [hunk 1]
```diff
@@ -1441,6 +1461,27 @@ def test_leave_group_as_member(self):
             GroupMembership.objects.filter(user=self.member, group=self.group).exists()
         )
 
+    def test_leave_group_completes_swipe_session(self):
+        event = SwipeEvent.objects.create(group=self.group, status="active")
+        event.completed_by.add(self.leader)
```
[the remark this answers is not in the document]
Comment (LaxmanSRawat) [reply backend/groups/tests.py]: @copilot apply changes based on [this feedback](https://github.com/gcivil-nyu-org/team2-mon-spring26/pull/208#discussion_r3150334244)

File: backend/groups/tests.py [hunk 2]
```diff
@@ -1441,6 +1461,27 @@ def test_leave_group_as_member(self):
...
+        event.refresh_from_db()
+        self.assertEqual(event.status, SwipeEvent.Status.COMPLETED)
+
+    def test_helper_zero_participants_completes_session(self):
+        from groups.views import check_and_complete_active_swipe_events
+
+        # Clear memberships to make total_participants = 0
+        GroupMembership.objects.filter(group=self.group).delete()
+        event = SwipeEvent.objects.create(group=self.group, status="active")
+
```
[the remark this answers is not in the document]
Comment (LaxmanSRawat) [reply backend/groups/tests.py]: @copilot apply changes based on [this feedback](https://github.com/gcivil-nyu-org/team2-mon-spring26/pull/208#discussion_r3150334264)

File: backend/groups/views.py [hunk 3]
```diff
@@ -545,6 +545,64 @@ def api_invitation_action(request, invitation_id, action):
...
+
+    for event in active_events:
+        completed_count = event.completed_by.count()
+        if completed_count >= total_participants:
+            all_swipes = event.swipes.all()
+            threshold = math.ceil(total_participants * 2 / 3)
+            venue_likes = (
+                all_swipes.filter(direction=Swipe.Direction.RIGHT)
+                .values("venue_id")
+                .annotate(like_count=Count("user_id", distinct=True))
```
[the remark this answers is not in the document]
Comment (LaxmanSRawat) [reply backend/groups/views.py:579]: @copilot apply changes based on [this feedback](https://github.com/gcivil-nyu-org/team2-mon-spring26/pull/208#discussion_r3150334287)

**Review by LaxmanSRawat — APPROVED** [review]
